demo/src/app/pages/demo/components/advanced-example-types.component.ts

```diff
@@ -14,77 +15,88 @@ export class AdvancedExamplesTypesComponent {
       name: "Leanne Graham",
       username: "Bret",
       email: "Sincere@april.biz",
-      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu."
+      comments: "Lorem ipsum dolor sit amet, ex dolorem officiis convenire usu.",
+      custom: '10|20'
```

Comment:
@damnko sorry, but probably we don't need this feature to update this PR? As most likely this will add up to the example complexity which we want to avoid? Probably we can change the example itself (as per this comment https://github.com/akveo/ng2-smart-table/pull/152#discussion_r99551738) so that we won't need this feature under this PR?